src/ontology/profile.txt

```diff
@@ -1,25 +1 @@
-WARN	annotation_whitespace
-ERROR	deprecated_boolean_datatype
-ERROR	deprecated_class_reference
-ERROR	deprecated_property_reference
-ERROR	duplicate_definition
-WARN	duplicate_exact_synonym
-WARN	duplicate_label_synonym
-ERROR	duplicate_label
-WARN	duplicate_scoped_synonym
-WARN	equivalent_pair
-WARN	invalid_xref
-ERROR	label_formatting
-ERROR	label_whitespace
-INFO	lowercase_definition
-WARN	missing_definition
-ERROR	missing_label
-WARN	missing_obsolete_label
-ERROR	missing_ontology_description
-ERROR	missing_ontology_license
-ERROR	missing_ontology_title
-INFO	missing_superclass
-ERROR	misused_obsolete_label
-ERROR	multiple_definitions
-ERROR	multiple_equivalent_classes
-ERROR	multiple_labels
+INFO	invalid_xref
```

Comment:
## The changes
Nico did this in [this commit](https://github.com/monarch-initiative/mondo-ingest/pull/600/commits/a620022faf9dddf6d8965e3afecbc11721257259), as part of #600. But I felt it might help to have this show up as its own PR.